openapi.yaml

```diff
@@ -16,60 +16,343 @@ host: api.oregonstate.edu
+        - $ref: '#/parameters/gameId'
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/GameResult'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    patch:
+      tags:
+        - games
+      summary: Update a game specified by its id
+      operationId: patchGameByGameId
+      consumes:
+        - application/json
+      parameters: 
+        - $ref: '#/parameters/gameId'
+        - name: updateBody
+          in: body
+          description: Object that will overwrite the game specified by its id
+          required: true
+          schema:
+            $ref: '#/definitions/GamePatchBody'
+      responses:
+        '200':
+          description: Successfully update a game
+          schema:
+            $ref: '#/definitions/GameResult'
         '400':
-          description: Bad request
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/Errors'
+        '404':
+          description: Game not found
           schema:
             $ref: '#/definitions/Errors'
         '500':
           description: Internal server error
           schema:
             $ref: '#/definitions/Errors'
-  /pets/{id}:
+    delete:
+      tags:
+        - games
+      summary: "Remove a game specified by its id, also the participation of member in the game would be removed."
+      operationId: deleteGameByGameId
+      parameters:
+        - $ref: '#/parameters/gameId'
+      responses:
+        '204':
+          description: game successfully removed
+        '404':
+          description: No game with this id
+          schema:
+            $ref: "#/definitions/Error"
+        '500':
+          description: Internal server error
+          schema:
+            $ref: "#/definitions/Error"
+  /games/{gameId}/players:
     get:
-      summary: Info for a specific pet
+      summary: A list of player in the game with their specific in-game information
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - players
+      operationId: getPlayersByGameId
       parameters:
         - $ref: '#/parameters/authorization'
-        - name: id
-          in: path
-          description: Unique Pet ID
+        - $ref: '#/parameters/gameId'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/PlayerInGameResults'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /games/{gameId}/players/{playerId}:
```

Comment:
I think this should be `memberId`. There's a few other places that this will need to be changed also